superset/tags/schemas.py

```diff
@@ -58,7 +58,7 @@ class TagObjectSchema(Schema):
     name = fields.String()
     description = fields.String(required=False, allow_none=True)
     objects_to_tag = fields.List(
-        fields.Tuple((fields.String(), fields.Int())), required=False
+        fields.Tuple((fields.String(), fields.Int())),
```

Comment:
does this change the api? I know this is behind a feature flag, but in your opinion, is this a breaking change?

Reply:
good call out, this is a breaking change it's prolly safer to just keep as not required i've already have the proper .get() default value in place